tests/flows/validation/test_transfer.py

```diff
@@ -435,6 +440,7 @@ def test_validate_transfer_no_items(
     assert param["error"] == ERR_ADOBE_MEMBERSHIP_ID_EMPTY.to_dict()
     assert param["constraints"]["hidden"] is False
     assert param["constraints"]["required"] is True
+    assert not get_product_items_by_skus_mock.called
```

Comment:
I would prefer here to use function `assert_not_called`

Reply:
Done!